pypeman/commands.py

```diff
@@ -27,6 +27,12 @@
 import warnings
 from functools import partial
 
+# TODO: can remove try except if we have DaemonLite as a hard requirement
+try:
+    from DaemonLite import DaemonLite
```

Comment:
We can make it a hard requirements for now so you can remove this try/except.

Reply:
removed try except